### PR TITLE
Development

### PR DESCRIPTION
**Pull request by jgbowser**
Built out a mobile first view of the hero section and hero content. Added some basic responsive styling expecting it to likely be adjusted as we settle on a final hero image. Let me know if you see any issues.

### CODE REVIEW

File: src/components/HeroBox.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import heroBoxStyles from '../styles/hero-box.module.css'
+import logo from '../images/utahjs-logo.png'
+
+const HeroBox = () => {
+  return (
+    <div className={heroBoxStyles.hero_box}>
+      <div className={heroBoxStyles.hero_box_inner}>
+        <img className={heroBoxStyles.logo} src={logo} alt='UtahJS Logo' />
+        <h2>UtahJS</h2>
+        <div className={heroBoxStyles.hero_subtext}>
```
Comment (briancbarrow) [inline src/components/HeroBox.js:12]: This should be a `p` tag just so it's semantic.

**Review by briancbarrow — CHANGES_REQUESTED** [review]
Update `div`s to `p`s. Also rename the svg file

**Review by briancbarrow — APPROVED** [review]

File: gatsby/src/components/Hero.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+import HeroBox from './HeroBox'
+import heroStyles from '../styles/hero.module.css'
```
Comment (mikelnorth) [inline gatsby/src/components/Hero.js:4]: Lets move this styled components

File: gatsby/src/components/HeroBox.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import heroBoxStyles from '../styles/hero-box.module.css';
```
Comment (mikelnorth) [inline gatsby/src/components/HeroBox.js:3]: same as above

File: gatsby/src/components/layout.css [hunk 4]
```diff
@@ -0,0 +1,605 @@
+@import url('https://fonts.googleapis.com/css2?family=Fira+Sans&family=Sanchez&display=swap');
```
Comment (mikelnorth) [inline gatsby/src/components/layout.css:1]: Delete this file

File: gatsby/src/styles/hero-box.module.css [hunk 5]
```diff
@@ -0,0 +1,79 @@
+.hero_box {
```
Comment (mikelnorth) [inline gatsby/src/styles/hero-box.module.css:1]: Move to styled component

**Review by mikelnorth — CHANGES_REQUESTED** [review]

File: gatsby/src/components/HeroBox.js [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+      <img className="logo" src={logo} alt="UtahJS Logo" />
+      <h2>UtahJS</h2>
+
+      <p className="hero_subtext">JavaScript Engineers of Utah</p>
+      <p className="hero_subtext">Let's learn together.</p>
+
+      <a className="btn yellow" href="https://conf.utahjs.com">
+        Conference
+      </a>
+      <a className="btn green" href="https://slack.utahjs.com">
```
Comment (mikelnorth) [inline gatsby/src/components/HeroBox.js:96]: import Button from 'react-bootstrap/button'

<Button href=''>Conferene</Button>

**Review by mikelnorth — APPROVED** [review]

**Pull request merged**